tool/src/webapp/js/edit-page.js

```diff
@@ -0,0 +1,43 @@
+var deleteButton;
+var insist = false;
+$(function() {
+    $('#delete-confirm').dialog({
+        autoOpen: false,
+        resizable: false,
+        modal: true,
+        dialogClass: "no-close",
+            buttons: [{text:"Delete",
+                      click: function() {
+                      insist = true;
+                      deleteButton.click();
+                  }},{text:"Cancel",
+                      click: function() {
+                      $( this ).dialog( "close" );}}
+    ]});
+    $("#grouplist").hide();
+    if ($('#grouplist input').size() == 0) {
+	$('#editgroups').hide();
+    }
+    $("#editgroups").click(function() {
+         $("#grouplist").show();
+         $("#editgroups").hide();
+    });
+    $('#delete-text-item').click(function(event) {
+        deleteButton = $(this);
+        var confirmMsg = document.getElementById('confirmDeleteLangSupp').innerHTML;
+        return delete_confirm(event, confirmMsg);
+    });
+});
+
+function delete_confirm(event, message) {
```

Comment:
I wouldn't include show-page.js in edit-page.html but maybe start a common file which is used across both?